### DISCUSSION TITLE
Переработка камня в железо и сталь

### DISCUSSION THREAD

**Discussion by PetrBronzo** [octothorp-team/public-dobrograd#729] [Предложения]
Пеработка камня в железо а железо в сталь

В шахте любой шахтер на ЦДБГ собирает 70% камня и только 30% полезных ископаемых (Железо, Сталь, Золото, Серебро, Драгоценности) Камень являясь бесполезным просто выкидывается, предлагаю решение: В очиститель добавить рецепт железа 8 камней = 1 железо и 8 железа = 1 сталь

Подобное решение решит вопрос с камнем и сделает Сталь более дешевле так как не все токари согласны покупать Сталь на рынке за 3000 руб прошу игроков к голосованию.

[poll type=regular results=always chartType=bar]
* ДА!, это создаст спрос на камень и решит вопрос с высокой ценой стали
* НЕТ!, меня устраивает нынешняя система добычи руды
[/poll]

**Comment by PetrBronzo** [comment]
> Пеработка камня в железо а железо в сталь
> 
> В шахте любой шахтер на ЦДБГ собирает 70% камня и только 30% полезных ископаемых (Железо, Сталь, Золото, Серебро, Драгоценности) Камень являясь бесполезным просто выкидывается, предлагаю решение: В очиститель добавить рецепт железа 8 камней = 1 железо и 8 железа = 1 сталь
> 
> Подобное решение решит вопрос с камнем и сделает Сталь более дешевле так как не все токари согласны покупать Сталь на рынке за 3000 руб прошу игроков к голосованию.
> 
> * ДА!, это создаст спрос на камень и решит вопрос с высокой ценой стали
> * НЕТ!, меня устраивает нынешняя система добычи руды

**Comment by SnakeLuck** [comment]
Лично по мне - это не лучшая идея, это не логично и слишком муторно
